pipeline/data_retrieval/retrieve_data.py

```
@@ -7,22 +8,26 @@
 import base64
 import sys
 import os
-from spark_kafka import produce_messages, add_to_es_topic
+from spark_kafka import produce_messages
+from langdetect import detect_langs, LangDetectException
+import random
```

Ruff (0.8.2)

Redefinition of unused `random` from line 7

Remove definition: `random`

(F811)